### PR TITLE
OKTA-614327 Enable passwordless auth for vanilla redirect app

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added passwordless authentication procedure to the vanilla JS redirect guide: https://developer.okta.com/docs/guides/auth-js-redirect/main/
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-614327](https://oktainc.atlassian.net/browse/OKTA-614327)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md [hunk 1]
```diff
@@ -303,6 +303,35 @@ Test the new configurations by recovering a password for a user of your sample a
...
 
+## Enable passwordless authentication
+
+Enable passwordless authentication for your existing users by configuring your Okta org's authenticator enrollment policy, authentication policy, and global session policy. This example uses the email authenticator to authenticate your users instead of a password. For full details and other passwordless implementation options, see [Set up passwordless sign-in experience](https://help.okta.com/okta_help.htm?type=oie&id=ext-passwordless).
+
+1. Go to **Directory** > **Groups** and click **Add group**. Give the group a name, for example, Passwordless Users, and click **Save**.
+1. Select your new group, and click **Assign people** from the **People** tab. Add one or more users to your new group.
+1. Go to **Security** > **Authenticators** and edit or ensure the **Email** authenticator is set to **Authentication and recovery**.
+1. Click the **Enrollment** tab and then click **Add a policy** to add a new enrollment policy targeted at your new group. Configure the following fields and then click **Create Policy**:
+    * **Policy name**: Any name for this policy, for example, Passwordles Enrollment
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md:314]:
```suggestion
    * **Policy name**: Any name for this policy, for example, Passwordless Enrollment
```

File: packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md [hunk 2]
```diff
@@ -303,6 +303,35 @@ Test the new configurations by recovering a password for a user of your sample a
...
+    * **Assign to Groups**: Your new group, Passwordles Users
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md:315]:
```suggestion
    * **Assign to Groups**: Your new group, Passwordless Users
```

File: packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md [hunk 3]
```diff
@@ -303,6 +303,35 @@ Test the new configurations by recovering a password for a user of your sample a
...
+    * **Email** authenticator: Set to required
+    * **Password** authenticator: Set to disabled
+1. Add a rule name, for example, Passwordless Enrollment Rule, and click **Create rule** to complete the enrollment policy setup.
+1. Click **Authentication Policies** and assign your sample application to a one-factor authentication policy, if it's not already. In the policy, click **Add rule**, and make the following configurations and then click **Save**:
+    * **Rule name**: Any name for this rule, for example, Passwordless authentication rule
+    * **User's group membership includes**: Your new group, Passwordless Users
+    * **User must authenticate with**: Any 1 factor type
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md:322]: The UI (for me) says the following:
```suggestion
    * **User must authenticate with**: Any 1 factor type / IdP
```

**Review by brentschaus-okta — APPROVED** [review]
LGTM

Just a few nits

File: packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md:314]: Ugh, nice catch.

File: packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md (hunk 3 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md:322]: thanks, I was using my preview org.

File: packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md [hunk 4]
```diff
@@ -303,6 +303,35 @@ Test the new configurations by recovering a password for a user of your sample a
...
+1. Select your new group, and click **Assign people** from the **People** tab. Add one or more users to your new group.
+1. Go to **Security** > **Authenticators** and edit or ensure the **Email** authenticator is set to **Authentication and recovery**.
+1. Click the **Enrollment** tab and then click **Add a policy** to add a new enrollment policy targeted at your new group. Configure the following fields and then click **Create Policy**:
+    * **Policy name**: Any name for this policy, for example, Passwordles Enrollment
+    * **Assign to Groups**: Your new group, Passwordles Users
+    * **Email** authenticator: Set to required
+    * **Password** authenticator: Set to disabled
+1. Add a rule name, for example, Passwordless Enrollment Rule, and click **Create rule** to complete the enrollment policy setup.
+1. Click **Authentication Policies** and assign your sample application to a one-factor authentication policy, if it's not already. In the policy, click **Add rule**, and make the following configurations and then click **Save**:
+    * **Rule name**: Any name for this rule, for example, Passwordless authentication rule
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md:320]: None. Just a miss. I'll title it. thanks.

**Pull request merged**